packages/schematics/src/add/index.spec.ts

```diff
@@ -58,15 +57,6 @@ describe('ng add @clr/angular', () => {
       const styles = configFile.projects[PROJECT_NAME].architect.build.options.styles;
 
       expect(styles.includes('node_modules/@clr/ui/clr-ui.min.css')).toBeTruthy();
```

Comment:
Do we want to verify that the Core styles are loaded too?

https://clarity.design/get-started/developing/#_2-global-styles

Reply:
I don't know. It was an honest question.

Reply:
I think adding these should be part of the implementation of `ng add @cds/angular` - the cds-scoped package. But I might be missing something. Is there a reason why we would want to add them with `@clr/angular`?

Reply:
You would know best. I have no preference or information that would contradict your own. I was just curious.

Reply:
@mathisscott, Github marks the review as `Changes requested`. Could you please approve the PR if everything's alright with it?